packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -403,13 +405,16 @@ export class List extends BaseComponent<IListProps, IListState> implements IList
     const pageStyle = this._getPageStyle(page);
 
     const { onRenderPage = this._onRenderPage } = this.props;
+    const pageRef = React.createRef<HTMLDivElement>();
```

Comment:
Hmm. What about this:

```ts
const pageRef = this._pageRefs[page.key] =  this._pageRefs[page.key] || React.createRef();
```

That way rather than creating a new ref object every time, you reuse the ref object you previously had.

Reply:
Great idea, especially since this is in a pretty tight loop. Thank you 🙇